pages/common/htpasswd.md

```diff
@@ -22,3 +22,7 @@
 - Verify user password:
 
 `htpasswd -v {{path/to/file}} {{user_name}}`
+
+- Generate username/password string (md5) for manual use:
```

Comment:
I'm not sure I understood this description. Does the command concatenate usarname+password then hashes it?

Reply:
No, it will output the line that is normally written into the htpasswd file, e.g:

`myusername:$apr1$MlN143435.A9jJJQrd17ABD9LIt.`

Context: I use this all the time, as my server's control panel demands complex passwords, but you don't want complex passwords all the time (eg if it's not security related but just to prevent a site from being reached by everybody pre-launch). So I generate the content of htpassword file by hand and paste it.

Maybe better like this?:

`Generate username (plain text) and password (md5) to paste into passwd file`

Reply:
This does read much better to me. Thank you for the explanation 👍

Reply:
`Output string` is not grammatically correct. May be "Display a string .."

Reply:
Or "return a string"